src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -158,12 +158,20 @@ static inline int32_t _pcm_shift(int32_t d, int32_t a)
 
 #define PCM_ABS32(x) (((x) ^ ((x) >> 31)) + (int)((unsigned int)(x) >> 31))
 
-static int32_t _pcm_convert_f_to_i(int32_t src)
+/**
+ * \brief convert float number to fixed point
+ *
+ * \param src integer number to convert
+ * \param pow additional exponent component, default 0
+ *            number of fractional bits in fixed point value - default 0
```

Comment:
C doesn't support default parameters. Can you say "Use '0' for normal conversion to integers" or something?

Reply:
changed

Reply:
Missed the instance in _pcm_convert_i_to_f.

Reply:
ok